Headlines/src/DataSources/FilterSourcesDataSource.swift

```diff
@@ -60,48 +60,48 @@ class FilterSourcesDataSource: NSObject, UICollectionViewDelegateFlowLayout, UIC
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
         return sources.count
     }
-    
+
     func collectionView(_ collectionView: UICollectionView,
                         cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        
+
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "cell", for: indexPath)
             as! LabelCollectionViewCell
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f5a5ab936ee0d164c160af7374b637afef6eb821/Headlines/src/DataSources/FilterSourcesDataSource.swift#L68"></span>Force casts should be avoided.<br />
<code>force_cast</code> <code>FilterSourcesDataSource.swift:68</code></td>
    </tr>
  </tbody>
</table>